creator-node/src/utils/index.ts

```diff
@@ -33,7 +33,8 @@ import { runShellCommand } from './runShellCommand'
 import { currentNodeShouldHandleTranscode } from './contentNodeUtils'
 import { clusterUtils } from './clusterUtils'
 
-export type { ValuesOf, RequestWithLogger } from './utils'
+export type { ValuesOf } from './utils'
+export type { CustomRequest } from '../apiHelpers'
```

Comment:
This is an odd export since it's exporting something that's not in the module. Doesn't break anything so it's fine, just feel a little odd

Reply:
i think this works bc it's defined in apiHelpers here https://github.com/AudiusProject/audius-protocol/pull/4180/files#diff-46283d16f78e9ccc2b121910ab8e3d74d30f1d9cf35001c96228af11c5670314R18-R21

Reply:
removed!

Reply:
dope